android/src/app/organicmaps/editor/AdvancedTimetableFragment.java

```diff
@@ -137,7 +143,21 @@ private static void setTextChangedListener(@Nullable EditText input,
       public void beforeTextChanged(CharSequence s, int start, int count, int after) {}
 
       @Override
-      public void onTextChanged(CharSequence s, int start, int before, int count) {}
+      public void onTextChanged(CharSequence s, int start, int before, int count) {
+        // Check if valid
+        // If not valid
+        // show error on textview
+        // Disable and grey out save button
```

Comment:
This comment doesn't look good. Please use clear English sentences where necessary.

Reply:
Removed.